draft-pda-protocol.md

```diff
@@ -540,24 +549,20 @@ To encrypt an input share, the client first generates an HPKE
 {{!I-D.irtf-cfrg-hpke}} context for the aggregator by running
 
 ~~~
-enc, context = SetupBaseS(pk, [TODO])
+enc, context = SetupBaseS(pk, "pda input share" || server_role)
```

Comment:
Do we have to define "||" as concatenation somewhere?

Reply:
I don't think so.